tests/libvmi/vmi.go

```diff
@@ -38,6 +38,8 @@ type Option func(vmi *v1.VirtualMachineInstance)
 func New(opts ...Option) *v1.VirtualMachineInstance {
 	vmi := baseVmi(randName())
 
+	WithResourceMemory(qemuMinimumMemory())(vmi)
```

Comment:
It does break some instance type tests though - https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/11366/pull-kubevirt-e2e-k8s-1.29-sig-compute/1763152795386515456

@EdDev - maybe it does make sense for a new factory function along the lines of `NewTiniest`?

Reply:
I do not think so, it makes little sense.
You could set whatever you need for Arm specifically, but in the end this just reflects a problem of the API.

Can you define the in-guest memory without touching the other? I think it should have an effect on the resource-memory in the end.

BTW, I think this fits now to register a default instead of setting it directly here:
https://github.com/kubevirt/kubevirt/pull/10975
But we can move it after that one gets merged.

Reply:
@EdDev kubevirt API has three incompatible ways to specify VMI memory. For example if you specify the guest memory, the memory request is calculated based on it, but not if you set it explicitly to say `1Mi`.
I don't think it would be easy for `libvmi` to hide this fact. It must make it possible to keep the memory request as nil. I don't see a way around adding a `NewTiny` to be used only by tests.